test/fixtures/rules/std_rule_cases/L045.yml

```diff
@@ -248,3 +248,23 @@ test_fail_query_uses_templating:
 
     SELECT var_bar
     FROM {{ ref('issue_2235') }}
+
+test_pass_update_cte:
```

Comment:
Think we could add a similar "failure" test case?

Reply:
Done.